web/public/mgmt/config.yml

```diff
@@ -2110,7 +2110,7 @@ collections:
       - { label: "Filename", name: "filename", widget: "write-once-read-only" }
       - { label: "Display Name", name: "displayname", widget: "nospace" }
       - { label: "Name", name: "name", widget: "string" }
-      - { label: "Id", name: "id", widget: "write-once-read-only" }
+      - { label: "Id", name: "id", widget: "write-once-read-only", required: true }
```

Comment:
The required flag is fine here.. the issue is that this field doesn't exist in another collection... I believe this is the right collection - License Tasks (Navigator with Webflow mappings)